tests/rptest/tests/partition_movement_test.py

```diff
@@ -343,23 +260,25 @@ def test_bootstrapping_after_move(self, num_to_upgrade):
             install_previous_version=test_mixed_versions,
             num_to_upgrade=num_to_upgrade)
         self.start_redpanda(num_nodes=3, install_opts=install_opts)
-        spec = TopicSpec(name="topic", partition_count=3, replication_factor=3)
-        self.client().create_topic(spec)
-        self.topic = spec.name
-        self.start_producer(1)
-        self.start_consumer(1)
-        self.await_startup()
+        topic = "topic"
+        self.rpk_client().create_topic(topic, 3, 3)
+        rw_verifier = RWVerifier(self.test_context, self.redpanda)
+        rw_verifier.start()
+        rw_verifier.remote_start(topic, self.redpanda.brokers(), topic, 3)
+        self.logger.info(f"waiting for 100 r&w operations to warm up")
         # execute single move
         self._move_and_verify()
-        self.run_validation(enable_idempotence=False, consumer_timeout_sec=45)
-
-        # snapshot offsets
-        rpk = RpkTool(self.redpanda)
+        self.logger.info(f"waiting for 100 r&w operations")
+        rw_verifier.ensure_progress(topic, 100, 10)
+        self.logger.info(f"checkin if cleared 1000 operations")
+        rw_verifier.has_cleared(topic, 1000, 45)
+        rw_verifier.remote_stop(topic)
+        rw_verifier.remote_wait(topic)
```

Comment:
this test is not equivalent as the verifier doesn't use consumer groups, would it be possible to add consumer groups support to verifier ?